public/index.html

```diff
@@ -0,0 +1,741 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Brand Marketing App</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="description" content="Project Week One">
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300,100,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto+Condensed:400,700,300' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Raleway:400,100,300,500,600' rel='stylesheet' type='text/css'>
+    <link rel='stylesheet' href='css/stylesheet.css' type='text/css'>
+  </head>
+  <body>
+    <section class="home" id="home">
+      <section class="a1-home-header">
+        <section class="appbox-logo-section">
+          <img class= "appbox-logo" src="./img/appbox.png" alt="appbox logo">
+        </section>
+        <nav class="home-nav">
+          <ul>
+            <li><a class="home-nav-a" href="#home">HOME</a></li>
+            <li><a class="home-nav-a" href="#about">ABOUT</a></li>
+            <li><a class="home-nav-a" href="#features">FEATURES</a></li>
+            <li><a class="home-nav-a" href="#reviews">REVIEWS</a></li>
+            <li><a class="home-nav-a" href="#screens">SCREENS</a></li>
+            <li><a class="home-nav-a" href="#blog">BLOG</a></li>
+            <li><a class="home-nav-a" href="#support">SUPPORT</a></li>
+            <li class="icon-seek"></li>
+          </ul>
+        </nav>
+      </section>
+      <section class="a2-home-top">
+        <section class="home-h1-section">
+          <h1 class="home-h1 a2-h1">Best Way To Present Your<br> Beautiful Mobile App</h1>
+          <p class="home-p home-p-first-sentance">AppBox is full of awesome features. It's designed for</p>
+          <p class="home-p home-p-second-sentance"><strong>App Landing and App Store site</strong></p>
```

Comment:
Same typo here (`sentance` should be `sentence`).

For this -- rather than break it out in to two `p` elements, this should be one. You have a single sentence that you're telling the document is actually two paragraphs, which is not correct. The sentence being broken up here is "It's designed for App Landing and App Store site" - a more appropriate handling of this might be something like this:

```html
<p class="intro-copy">AppBox is full of awesome features. It's designed for <strong>App Landing and App Store site</strong></p>
```

Then you have the ability to style your paragraph like this:

```Sass
.intro-copy {
  font-size: 0.75em;

  strong {
    font-size: 1.5em;
  }
}
```

This way your markup is both more resilient to changes, but also a better representation of the content being shown.